roles/basket-proxy/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: configure nginx upstream
+  sudo: true
+  template: src=upstream.conf.j2 dest=/etc/nginx/conf.d/upstream/http_fxa_basket_proxy.conf
+  notify: reload nginx config
+
+- name: configure nginx location
+  sudo: true
+  template: src=nginx.conf.j2 dest=/etc/nginx/conf.d/location/http_fxa_basket_proxy.conf
+  notify: reload nginx config
+
```

Comment:
I'm not clear on why this nginx proxy config is here since this basket proxy should be bound to localhost and is inaccessible via the ELB.